browser-interface/packages/shared/apis/host/Players.ts

```diff
@@ -92,11 +92,7 @@ export function registerPlayersServiceServerImplementation(port: RpcServerPort<P
     },
     async getConnectedPlayers() {
       return {
-        players: Array.from(getAllPeers().values())
-          .filter(($) => $.visible)
-          .map((peer) => {
-            return { userId: peer.ethereumAddress }
-          })
+        players: getPeerEthereumAddresses()
```

Comment:
It was important for the logic of the scenes that it only returns the visible avatars. Does `getPeerEthereumAddresses` inherit that logic? I can't tell by the name
```suggestion
        players: getPeerEthereumAddresses()
```

Reply:
Fixed